jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPartCompliance.java

```diff
@@ -70,6 +91,85 @@ public static MultiPartCompliance valueOf(String name)
         return null;
     }
 
+    /**
+     * Create compliance set from a set of allowed Violations.
+     *
+     * @param violations A string of violations to allow:
+     * @return the compliance from the string spec
+     */
+    public static MultiPartCompliance from(Set<MultiPartCompliance.Violation> violations)
+    {
+        return new MultiPartCompliance("CUSTOM" + __custom.getAndIncrement(), violations);
+    }
+
+    /**
+     * Create compliance set from string.
+     * <p>
+     * Format: &lt;BASE&gt;[,[-]&lt;violation&gt;]...
```

Comment:
Use {@code ...} so you can avoid HTML entities.

Reply:
Done.